includes/admin/class.llms.admin.settings.php

```diff
@@ -935,6 +935,7 @@ public static function get_option( $option_name, $default = '' ) {
 	 * @since 3.35.2 Don't strip tags on editor and textarea fields that allow HTML.
 	 * @since 5.9.0 Stop using deprecated `FILTER_SANITIZE_STRING`.
 	 * @since [version] Add handling for array fields for standard input types.
+	 *               Account for the `maxlength` input text and textarea attribute.
```

Comment:
```suggestion
	 * @since 3.35.2 Don't strip tags on editor and textarea fields that allow HTML.
	 * @since 5.9.0 Stop using deprecated `FILTER_SANITIZE_STRING`.
	 * @since [version] Add handling for array fields for standard input types.
	 *              Account for the `maxlength` input text and textarea attribute.
```